### PR TITLE
Space Error for no auth

### PR DESCRIPTION
**Pull request by adamweeks**
This also checks `error.message` to give a proper error title.

### CODE REVIEW

**Comment by adamweeks** [comment]
This will fail until #212 merges.

**Comment by adamweeks** [comment]
Ready for review.

File: packages/node_modules/@ciscospark/widget-space/src/selector.js [hunk 1]
```diff
@@ -69,14 +56,52 @@ export const getActivityTypes = createSelector(
   }
 );
 
+const getWidgetError = createSelector(
+  [getWidget, getSpark, getFormatMessage],
+  (widget, spark, formatMessage) => { // eslint-disable-line complexity
+    const registerError = spark.getIn([`status`, `registerError`]);
+    const widgetError = widget.getIn([`status`, `isError`]);
```
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-space/src/selector.js]: Could be imagining that we talked about this, but didn't we say we wanted this to be `hasError`?

Comment (adamweeks) [inline packages/node_modules/@ciscospark/widget-space/src/selector.js]: Yea, I don't think it was this code, but we definitely wanted to use `hasError`. I'll change that.

**Comment by adamweeks** [comment]
Once the recents code gets tackled, I'll figure out how to refactor into reusable code bits, but leaving it like this for now.

**Pull request merged**